layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/uEnv.txt_internal

```diff
@@ -0,0 +1,4 @@
+setemmcroot=if mmc dev 0 ; then setenv mmcroot /dev/mmcblk1p2 rw ; else setenv mmcroot /dev/mmcblk0p2 rw ; fi;
```

Comment:
This needs some cleanup as it reintroduces dead variables:
https://github.com/resin-os/meta-resin/commit/22a811d1763e736b75ed05f826724ec5d48846a5